src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  let price = cost * days;
+
+  if (days >= 3 && days < 7) {
+    price -= 20;
+  }
+
+  if (days >= 7) {
+    price -= 50;
+  }
```

Comment:
Would be great to create variables for 3, 7, 20 and 50 to make your code more descriptive

Reply:
Done. Changed the logic a bit. First the function checks whether days are more that 7 and then if days are more than 3.
Thank you!